_events/sr2020/virtual-competition-league-1.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Virtual Competition League 1
```

Comment:
I realise this name is getting quite long, but maybe clarify that this is session one rather than a subset of the league? I read this just now like it was a group stage or something.

Reply:
Should it be _Virtual Competition League Session 1_? I wasn't sure how to name these

Reply:
I was wondering that, but it gets rather a mouthful.
For the compstate I've currently got "First Saturday league" etc., though I'm open to changing that if we can find a good name. I'd definitely like to have the events and the compstate be similar if we can.